database/redis/last_check.go

```diff
@@ -71,14 +75,20 @@ func (connector *DbConnector) getSelfStateCheckCountKey(isRemote bool) string {
 	return selfStateChecksCounterKey
 }
 
+func appendRemoveTriggerLastCheckToRedisPipeline(context context.Context, pipe redis.Pipeliner, triggerID string) redis.Pipeliner {
```

Comment:
optional: Variable name is same as package name. Maybe better change to "ctx"?

Reply:
Changed to `ctx`